### PR TITLE
API changes

### PR DESCRIPTION
**Pull request by antoinegrant**
I brought the API changes that @rickitan made in https://github.com/Shopify/theme-scripts/pull/96

### CODE REVIEW

**Comment by antoinegrant** [comment]
I need to update the README with the API changes.

**Comment by antoinegrant** [comment]
Ok the REAME has been updated. All good for a round of reviews.

File: packages/theme-predictive-search/README.md [hunk 1]
```diff
@@ -1,43 +1,47 @@
...
+    <td>The search query.</td>
+  </tr>
+  <tr>
+    <td><code>resources</code> (required)</td>
+    <td>Object</td>
+    <td>Requests <code>resources</code> results for the given query, based on the <code>type</code> and <code>limit</code> fields.</td>
+  </tr>
+  <tr>
+    <td><code>type</code> (required)</td>
+    <td>Comma-separated values</td>
```
Comment (carolineschnapp) [inline packages/theme-predictive-search/README.md]: We say comma-separated values here, but this is what is needed when building the request URL. The Predictive Search Library can receive an array for `fields` and `type` as it automatically transforms arrays into comma-separated lists when building the request URL.

File: packages/theme-predictive-search/README.md [hunk 2]
```diff
@@ -1,43 +1,47 @@
...
+    <td>Specifies <code>options</code> for the requested resources based on the <code>unavailable_products</code> and <code>fields</code> settings.</td>
+  </tr>
+  <tr>
+    <td><code>unavailable_products</code> (optional)</td>
+    <td>String</td>
+    <td>Specifies whether to display results for unavailable products. The three possible options are <code>show</code>, <code>hide</code>, and <code>last</code>. Set to <code>last</code> to display unavailable products below other matching results.  Set to <code>hide</code> to filter out unavailable products from the search results. Default: <code>last</code>.</td>
+  </tr>
+  <tr>
+    <td><code>fields</code> (optional)</td>
+    <td>Comma-separated values</td>
```
Comment (carolineschnapp) [inline packages/theme-predictive-search/README.md]: Same comment as above for `type`.

File: packages/theme-predictive-search/README.md [hunk 3]
```diff
@@ -55,7 +59,10 @@ var predictiveSearch = new PredictiveSearch({
     fuzzy: true,
```
Comment (BrendaSegalShopify) [inline packages/theme-predictive-search/README.md]: Remove `fuzzy: true`

File: packages/theme-predictive-search/README.md [hunk 4]
```diff
@@ -55,7 +59,10 @@ var predictiveSearch = new PredictiveSearch({
...
     types: [PredictiveSearch.TYPES.PRODUCT],
```
Comment (BrendaSegalShopify) [inline packages/theme-predictive-search/README.md]: replace `types:` with `type:`.

File: packages/theme-predictive-search/README.md (hunk 1 above)
Comment (antoinegrant) [reply packages/theme-predictive-search/README.md]: Good catch 👍

**Comment by antoinegrant** [comment]
Comments addressed! 👍

**Pull request merged**